public/locales/auset-te.json

```diff
@@ -0,0 +1,426 @@
+      "setupTitle": "వుడ్‌ల్యాండ్ అలయన్స్‌ను ఏర్పాటు చేయండి",
+      "setup": "<i>1.</i> 3 <strong>కార్డ్‌లను</strong> గీయండి మరియు వాటిని మీ మద్దతుదారుల స్టాక్‌కు ముఖంగా జోడించండి.<br/><i>2.</i> <strong>తో మీ సానుభూతి ట్రాక్‌ను పూరించండి సానుభూతి టోకెన్‌లు</strong>.<br/><i>3.</i> మీ మ్యాచింగ్ బేసెస్ స్పేస్‌లలో మీ 3 <strong>బేస్‌లను</strong> ఉంచండి."
+    },
+    "corvid": {
+      "name": "కోర్విడ్ కుట్ర",
+      "summaryTitle": "కార్విడ్స్ గా ఆడటం",
+      "summary": "<p>కోర్విడ్ కుట్రగా, వుడ్‌ల్యాండ్ యుద్ధం యొక్క ఆటుపోట్లను నియంత్రిస్తూ, తెర వెనుక ఉన్న నిజమైన శక్తి మీరేనని మీరు చూపించాలనుకుంటున్నారు. మీరు మ్యాప్‌లో ఉంచిన <strong>ప్లాట్</strong> టోకెన్‌ను మీరు తిప్పిన ప్రతిసారీ, మీరు పాయింట్‌లను స్కోర్ చేస్తారు. మీరు మ్యాప్‌లో ఎక్కువ ముఖాముఖి ప్లాట్‌లు చేసినప్పుడు, మీరు ఎక్కువ పాయింట్లను స్కోర్ చేస్తారు.</p><p>ప్లాట్‌లను ఉంచడానికి మరియు తిప్పడానికి, మీరు మీ యోధులను తెలివిగా నియమించుకోవాలి మరియు ఉంచాలి. మీ లక్ష్యం కోసం యోధులను నియమించుకోవడంలో మీ వర్గం అత్యుత్తమమైనది, కానీ మీరు చాలా తరచుగా కదలడానికి చాలా కష్టపడతారు కాబట్టి వారు చాలా సన్నగా ఉంటారు. ఇంకా ఎక్కువగా, మీరు చాలా ఎక్కువ యుద్ధం చేయలేరు, ఎందుకంటే ప్లాట్‌ను ఉంచడానికి మీరు ఒకటి లేదా అంతకంటే ఎక్కువ మంది కొర్విడ్ యోధులను తీసివేయవలసి ఉంటుంది మరియు ప్లాట్‌ను క్లియర్ చేయడంలో కార్విడ్ యోధుడు ఉంటే తప్ప మీరు దానిని తిప్పలేరు.</p><p>కృతజ్ఞతగా, మీ యోధులు చాలా <strong>చురుకైన</strong>వారు, పాలనతో సంబంధం లేకుండా వారిని తరలించడానికి వీలు కల్పిస్తారు. అదనంగా, మీ <strong>ఎంబెడెడ్ ఏజెంట్లు</strong> బ్రూట్ ఫోర్స్‌తో మీ ప్లాట్‌లను అణచివేయడం ప్రమాదకరం - ఫేస్‌డౌన్ ప్లాట్ టోకెన్‌తో యుద్ధంలో డిఫెన్స్ చేసినప్పుడు, మీరు అదనపు హిట్‌ని ఎదుర్కొంటారు.</p><p>అయితే, జాగ్రత్త వహించండి మీ ప్లాట్లను <strong>ఎక్స్‌పోజర్‌</strong>కి తెరిచి ఉంచడం. ఫేస్‌డౌన్ ప్లాట్‌తో క్లియరింగ్‌లో ఉన్న ఏదైనా శత్రు వర్గం అది ఏ ప్లాట్ అని అంచనా వేయడానికి మీకు సరిపోలే కార్డ్‌ని చూపుతుంది. వారు సరైన వారైతే, వారు మీలో చొరబడి మీ ప్లాట్‌ను విఫలం చేసారు! వారు ప్లాట్‌ను తీసివేసి, దాని ప్రభావాన్ని విస్మరిస్తారు. వారు తప్పు చేస్తే, మీరు వారి గూఢచారిని రెడ్ హ్యాండెడ్‌గా పట్టుకోండి. మీ ప్లాట్లు మిగిలి ఉన్నాయి మరియు వారు మీకు చూపించిన కార్డును మీకు అందిస్తారు. బాగా బ్లఫ్ చేయండి.</p>",
+      "setupTitle": "కోర్విడ్ కుట్రను సెటప్ చేయండి",
+      "setup": "<i>1.</i> మాతృభూమి క్లియరింగ్‌ను ఎంచుకోండి. మీకు నచ్చిన 1 <strong>యోధుడు</strong> మరియు 1 <strong>ప్లాట్ టోకెన్</strong>ని అక్కడ ఉంచండి.<br/><i>2.</i> 1 <strong>యోధుడిని</strong> ఉంచండి. > ప్రతి సూట్ క్లియరింగ్‌లో. <i>(మునుపటి దశను లెక్కించడం, మొత్తం 4 మంది యోధులను ఉంచండి.)</i>"
+    },
+    "cult": {
+      "name": "బల్లి కల్ట్(లిజార్డ్ కల్ట్)",
```

Comment:
I think the phonetic translation is a nice touch, however if the others are removed this should also be removed for consistency.